setup.py

```diff
@@ -1,19 +1,6 @@
 #!/usr/bin/env python
 """setup.py for theme."""
 
-import subprocess
-
 from setuptools import setup
-from setuptools.command.sdist import sdist
-
-
-class BuildPy(sdist):
-    """Build the theme on sdist."""
-
-    def run(self):
-        """Run the build process."""
-        subprocess.call(("python", "translate_templates.py"))
-        sdist.run(self)
-
 
-setup(use_scm_version={"write_to": "src/version.py"}, cmdclass={"sdist": BuildPy})
+setup(use_scm_version=True)
```

Comment:
this pulls the version info from this repo, right?

Reply:
Yes, if it's a git checkout it uses the nearest git tag. For the sdist, I think it gets it from the .egg-info or the PKG-INFO, but I'm not 100% sure.